src/module/actor/sheet/base.ts

```diff
@@ -254,7 +254,7 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
         this.activateInventoryListeners(inventoryPanel);
 
         // Item chat cards
-        for (const element of htmlQueryAll(html, ".item[data-item-id] .item-image, .item[data-item-id] .item-chat")) {
+        for (const element of htmlQueryAll(html, "[data-item-id] .item-image, [data-item-id] .item-chat")) {
```

Comment:
Attribute selectors aren't indexed in the DOM, so they should never be used in isolation.

Reply:
You'll have to restore item I believe then. Good point.